descholar-ceo/find-flight-info
# Redirect user when authenticated

#### What does this PR ​ do​ ?
It allows any authenticated user previously to be automatically redirected to his page

#### Description of Task to be completed?
* -Add function `handleRedirect` in handlers file to redirect user to his page

#### How should this be manually tested?

* In case you don't have this repository on your local computer, proceed from step [1]
* If you already have this repo, start from step, open this project's root directory in terminal, and start by step [2.ii.]

* [1] Clone this repository by `git clone https://github.com/descholar-ceo/find-flight-info`
* [2] Type these commands : 
    * [2.i.]  `cd find-flight-info` to go in the root directory of the project
    * [2.ii.] `git pull origin ft-landing-page` to pull this branch
    * [2.iii.] `npm install` or `yarn add` to install all the packages
    * [2.iv.] `npm start` or `yarn start`
* [3] Then go in the browser, you will see a default opened page a nice beautiful homepage which holds description of the application

#### Any background context you want to provide?
* You have to install [nodejs](https://nodejs.org/en/) version 6 or higher prior to test this application

#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-02-24 11-55-22](https://user-images.githubusercontent.com/56886064/75142836-946adc80-56fc-11ea-9483-8919c6302efd.png)


#### Questions:
No Question
